app/routes.py

```diff
@@ -1,2 +1,296 @@
+            customer.postal_code = form_data["postal_code"]
+            customer.phone = form_data["phone"]
+
+            db.session.commit()
+            return customer.to_json(), 200
+    
+    # DELETE a specific customer
+    elif request.method == "DELETE":
+        db.session.delete(customer)
+        db.session.commit()
+        return jsonify({
+            "id": customer.id,
+            "details": f'Customer {customer.name} successfully deleted'}), 200
+
+
+# <------------------------------------- VIDEO ENDPOINTS ------------------------------------->
+
+videos_bp = Blueprint("videos", __name__, url_prefix= "/videos")
+
+# endpoint retrieves details about ALL videos and creates a record of a video
+@videos_bp.route("", methods=["GET", "POST"], strict_slashes=False)
+def video_details():
+
+    # GET all existing videos and details about each video
+    
+    if request.method == "GET":
+
+        all_videos = Video.query.all()
+
+        videos_response = []
+        for video in all_videos:
+            videos_response.append({
+                "id": video.id,
+                "title": video.title,
+                "release_date": video.release_date,
+                "total_inventory": video.total_inventory,
+                "available_inventory": 0
+            })
+        
+        return jsonify(videos_response), 200
+
+    # CREATES (POST) a new VIDEO
+
+    elif request.method == "POST":
+        request_body = request.get_json()
+        if "title" not in request_body or "release_date" not in request_body or "total_inventory" not in request_body:
+            return jsonify({"details": "Invalid data"}), 400
+        else:
+            new_video = Video(
+                title=request_body["title"], 
+                release_date=request_body["release_date"], 
+                total_inventory=request_body["total_inventory"],
+                available_inventory=request_body["total_inventory"]
+                )
+
+            db.session.add(new_video)
+            db.session.commit()
+
+            return make_response({
+                "id": new_video.id,
+                "title": new_video.title
+                }, 201) 
+
+# endpoint retrieves, updates and deletes records of a specific video
+@videos_bp.route("/<video_id>", methods=["GET", "PUT", "DELETE"], strict_slashes=False)
+def specific_video(video_id):
+
+    # GET details about specific video
+
+    video = Video.query.get(video_id)
+
+    if video is None:
+        return make_response(), 404
+    
+    elif request.method == "GET":
+        return video.to_json(), 200
+
+    # UPDATE (PUT) & return details about specific video
+
+    elif request.method == "PUT":
+        form_data = request.get_json()
+
+        if "title" not in form_data or "release_date" not in form_data or "total_inventory" not in form_data:
+            return jsonify({"details": "Invalid data"}), 400
+
+        else:
+            video.title = form_data["title"]
```

Comment:
Similar to the comment above, consider removing the else to enhance readability.